### DISCUSSION TITLE
Issue with multiple 32bit float geotiff layers and transparency

### DISCUSSION THREAD

**Discussion by mikebrose** [pelicanmapping/osgearth#2002] [Support Q&A]
I realize that osgearth does not provide full support for 32bit float images.
I have two image layers using driver="gdal". 
Both layers are 32bit float geotiffs with data only in the red channel. 
I have an upper layer that is elevated to an altitude via vertex shader in the .earth file.
I also have a 32bit float ground geotiff. 
My problem:
If I explicity set the upper layer's alpha value via frag shader in the .earth file to a value like .5, I cannot view the ground geotiff through it. 
If I also include a ready map visual imagery layer: 
<url>http://readymap.org/readymap/tiles/1.0.0/7/</url>      driver="tms"
This is viewable through the upper geotiff.
Why is the upper layer geotiff completely obscuring the ground geotiff even with alpha .5, but appropriately blending with the readymap visual imagery. 

In the attached screenshot, the perspective is down towards the earth and slightly angled to the left, peeking around the right edge of the upper layer. The upper layer is farthest left, semi transparent, and elevated. The ground geotiff is darker red and black, in the middle, and the readymap visual imagery is everywhere else. This screenshot shows the upper layer obscuring the ground geotiff, but not the readymap.

Appreciate any advice or thoughts on this. I can paste the shader if I need to. Thanks!   

![Screenshot from 2022-02-14 14-30-29](https://user-images.githubusercontent.com/65251392/156053494-4dc0ceb7-f287-4a7a-9733-c586126d2b2f.png)

**Comment by gwaldron** [comment]
Can you reproduce this with a simple earth file that your post?
